docker-compose.yml

```diff
@@ -23,8 +23,8 @@ services:
     volumes:
       - ${APP_CODE_PATH_HOST}:${APP_CODE_PATH_CONTAINER}
       - ${DATA_PATH_HOST}/pbf:${APP_CODE_PATH_CONTAINER}/pbf
-      - ./maps-config/osm-bright.tm2source/imposm_mapping.yml:/etc/imposm/mapping.yml
-      - ./workspace/osm-initial-import.sh:/usr/bin/osm-initial-import
+      - ${PWD}/maps-config/osm-bright.tm2source/imposm_mapping.yml:/etc/imposm/mapping.yml
```

Comment:
I don't think we want to do this, it actually makes the file sensitive to current working directory when we want to keep it insensitive.  Relative paths like "./" take the docker-compose file as the starting point, so this should have been safe.